sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala

```diff
@@ -219,6 +201,44 @@ class ContinuousSuite extends ContinuousSuiteBase {
       StopStream)
   }
 
+  test("kill task") {
```

Comment:
this test does not verify killing tasks :) it verifies "task failure stops the query" or "task failure restarts the query"